website/client/components/notifications.vue

```diff
@@ -165,6 +165,31 @@ const NOTIFICATIONS = {
     label: ($t) => `${$t('achievement')}: ${$t('achievementBackToBasics')}`,
     modalId: 'generic-achievement',
   },
+  ACHIEVEMENT_PARTY_UP: {
+    achievement: true,
+    label: ($t) => `${$t('achievement')}: ${$t('achievementPartyUp')}`,
+    modalId: 'generic-achievement',
+  },
+  ACHIEVEMENT_PARTY_ON: {
+    achievement: true,
+    label: ($t) => `${$t('achievement')}: ${$t('achievementPartyOn')}`,
+    modalId: 'generic-achievement',
+  },
+  ACHIEVEMENT_BEAST_MASTER: {
```

Comment:
for the beast master, mount master and triad bing achievements we didn't have anything to show them on the website?

Reply:
No, we didn't show anything when people got that achievements. Just checked again on prod to make sure.